jsondata.go

```diff
@@ -2620,6 +2620,108 @@ var data = []byte(`{
         "W-SU",
         "Zulu"
     ],
+    "locales": [
```

Comment:
this list appears to implement WordPress-specific locale codes as opposed to the normative standards put forth by IETF and W3 in 
http://tools.ietf.org/html/bcp47 
https://www.w3.org/TR/ltli/

Golang already implements an API for these tags in the experimental packages here:
https://godoc.org/golang.org/x/text/language